chapter3/src/3_9_2/main.rs

```diff
@@ -0,0 +1,15 @@
+use std::fs::File;
+use std::io;
+use std::io::Write;
+
+fn main() -> io::Result<()> {
+    let mut bytes = [0; 1024];
+    for byte in bytes.iter_mut() {
+        *byte = rand::random()
+    }
```

Comment:
[nits] こう書くとスッキリしますし、パフォーマンスも良いです（1024回のループ程度で気にすることはないですが）。

```suggestion
    let bytes = rand::thread_rng().gen::<[u8; 1024]>();
```

Reply:
📝  なるほど。こんな書き方があるんですね！